ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -81,12 +75,6 @@ tests:
     env:
       OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
     workflow: openshift-upgrade-aws-sdn
-- as: e2e-aws-sdn-multi
```

Comment:
curious to why it's important to keep this job for 4.16? but yes, we do have a periodic.

Reply:
In theory, there are still customers using multitenant mode. And multitenant mode occasionally breaks in new releases and needs an update to https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/openshift-sdn/004-multitenant.yaml. (OK, I guess it hasn't needed an update since 4.10... but it still could end up needing an update in 4.16 if someone adds a new core operator.)